test/test_function_call.py

```diff
@@ -127,7 +127,68 @@ def get_number():
     assert user.execute_function(func_call)[1]["content"] == "42"
 
 
+@pytest.mark.asyncio
+async def test_a_execute_function():
```

Comment:
rename function to sync def test_async_execute_function():

Reply:
See comment below.